pom.xml

```diff
@@ -322,6 +323,11 @@
 			<!-- not included in main lib directory - only in subdirectory -->
 			<scope>provided</scope>
 		</dependency>
+		<dependency>
+			<groupId>com.portalmedia</groupId>
+			<artifactId>embarc</artifactId>
+			<version>${embarc.version}</version>
+		</dependency>
```

Comment:
This should be:

```xml
<scope>provided</scope>
```

And `lib/embarc` should contain a copy of the `embarc-0.2.jar` as well as any of that jars dependencies. For example, I just tried to run the tool and got the following:

```
2022-03-17 16:17:08 - ERROR - Fits:688 - Tool error processing file: 00266.dpx
javax/activation/MimetypesFileTypeMap
java.lang.NoClassDefFoundError: javax/activation/MimetypesFileTypeMap
	at com.portalmedia.embarc.parser.FileFormatDetection.getMimeType(FileFormatDetection.java:39) ~[embarc-0.2.jar:?]
	at com.portalmedia.embarc.parser.dpx.DPXFileListHelper.createDPXFileInformation(DPXFileListHelper.java:24) ~[embarc-0.2.jar:?]
	at edu.harvard.hul.ois.fits.tools.embarc.EmbARC.extractInfo(EmbARC.java:60) ~[fits-1.5.2-SNAPSHOT.jar:1.5.2-SNAPSHOT]
	at edu.harvard.hul.ois.fits.tools.ToolBase.run(ToolBase.java:286) ~[fits-1.5.2-SNAPSHOT.jar:1.5.2-SNAPSHOT]
	at java.lang.Thread.run(Unknown Source) ~[?:?]
Caused by: java.lang.ClassNotFoundException: javax.activation.MimetypesFileTypeMap
	at jdk.internal.loader.BuiltinClassLoader.loadClass(Unknown Source) ~[?:?]
	at jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(Unknown Source) ~[?:?]
	at java.lang.ClassLoader.loadClass(Unknown Source) ~[?:?]
	... 5 more
```

This indicates that, at the minimum, you'll need a copy of the `javax.activation-api-1.2.0.jar` in that directory too.

Reply:
Ah ok will do. What command or config are you running to produce that error?

Reply:
```
mvn -DskipTests clean package
docker build -f docker/Dockerfile -t fits .
docker run --rm -v `pwd`:/work:z fits -i 00266.dpx
```

Reply:
Just pushed an update, let me know if it works for ya

Reply:
Yep, it's working for me now

Reply:
Great! Thanks for the help